source/guides/helpers/forms.md

```diff
@@ -201,6 +203,140 @@ Developers can customize attack handling.
 </form>
 ```
 
+### Pass A Value
```

Comment:
@davydovanton All this description was kept out with intention. This is overwhelming at the first glance and it's subject to change.